components/wpt-results.html

```diff
@@ -230,18 +230,19 @@
           this._refreshDisplayedNodes()
         }
 
-        const testRuns = await Promise.all(
-          this.testRunSpecs.map(async testRun => {
-            const pieces = testRun.split('@')
-            const browser = pieces[0]
-            const sha = pieces.length > 1 ? pieces[1] : 'latest'
-            const url = `/api/run?browser=${browser}&sha=${sha}`
-            const response = await window.fetch(url)
-            return response.json()
-          }))
+        // Fetch the TestRun JSON if it's not dumped inline.
```

Comment:
This should go through quickly right now because the site is broken. However, is there a plan to avoid the "use this strategy, unless it doesn't work, then use that strategy" here?

Reply:
It's not 'if it doesn't work' - it's essentially an alternative ctor for the Polymer element.